src/ui/flutter_app/lib/widgets/game_page.dart

```diff
@@ -780,62 +782,62 @@ class _GamePageState extends State<GamePage>
   }
   */
 
-  onAutoReplayButtonPressed() async {
+  Future<void> onAutoReplayButtonPressed() async {
     Navigator.of(context).pop();
 
     await onTakeBackAllButtonPressed(pop: false);
     await onStepForwardAllButtonPressed(pop: false);
   }
 
-  onGameButtonPressed() {
+  void onGameButtonPressed() {
     showModalBottomSheet(
       //showDialog(
       context: context,
       backgroundColor: Colors.transparent,
-      builder: (BuildContext context) {
-        return Semantics(
-          label: S.of(context).game,
-          child: SimpleDialog(
-            backgroundColor: Colors.transparent,
-            children: <Widget>[
-              SimpleDialogOption(
-                child: Text(
-                  S.of(context).newGame,
-                  style: AppTheme.simpleDialogOptionTextStyle,
-                  textAlign: TextAlign.center,
-                ),
-                onPressed: onStartNewGameButtonPressed,
+      builder: (context) => Semantics(
+        label: S.of(context).game,
+        child: SimpleDialog(
+          backgroundColor: Colors.transparent,
+          children: <Widget>[
+            SimpleDialogOption(
+              onPressed: onStartNewGameButtonPressed,
+              child: Text(
+                S.of(context).newGame,
+                style: AppTheme.simpleDialogOptionTextStyle,
+                textAlign: TextAlign.center,
               ),
-              SizedBox(height: AppTheme.sizedBoxHeight),
-              SimpleDialogOption(
-                child: Text(
-                  S.of(context).importGame,
-                  style: AppTheme.simpleDialogOptionTextStyle,
-                  textAlign: TextAlign.center,
-                ),
-                onPressed: onImportGameButtonPressed,
+            ),
+            const SizedBox(height: AppTheme.sizedBoxHeight),
+            SimpleDialogOption(
+              onPressed: onImportGameButtonPressed,
+              child: Text(
+                S.of(context).importGame,
+                style: AppTheme.simpleDialogOptionTextStyle,
+                textAlign: TextAlign.center,
+              ),
+            ),
+            const SizedBox(height: AppTheme.sizedBoxHeight),
+            SimpleDialogOption(
+              onPressed: onExportGameButtonPressed,
+              child: Text(
+                S.of(context).exportGame,
+                style: AppTheme.simpleDialogOptionTextStyle,
+                textAlign: TextAlign.center,
               ),
-              SizedBox(height: AppTheme.sizedBoxHeight),
+            ),
+            const SizedBox(height: AppTheme.sizedBoxHeight),
+            if (Config.screenReaderSupport)
               SimpleDialogOption(
                 child: Text(
-                  S.of(context).exportGame,
+                  S.of(context).close,
```

Comment:
I don't know why line 832 to line 879 was changed. It looks like it would affect the functionality. It would be great if this change could be extracted and made into a separate Git commit with a detailed explanation of the reason for the difference. If it's more of a hassle, you can reply with the reason for the change. Thanks!